src/deye_events.py

```diff
@@ -68,3 +98,32 @@ def get_id(self):
     @abstractmethod
     def process(self, events: list[DeyeEvent]):
         pass
+
+
+def events_to_str(events: list[DeyeEvent] | set[DeyeEvent]) -> str:
```

Comment:
Please make this function a static method of `DeyeEvent` class

Reply:
this is now a `__str__` method of `DeyeEventList`